examples/gno.land/r/demo/profile/handlers.gno

```diff
@@ -0,0 +1,154 @@
+package profile
+
+import (
+    "bytes"
+    "gno.land/p/demo/ufmt"
+    "gno.land/p/demo/mux"
+    "strings"
+    "std"
+)
+
+const (
+    BaseURL           = "/r/demo/profile"
+    SetStringFieldURL = BaseURL + "?help&__func=SetStringField&field=%s"
+    SetIntFieldURL    = BaseURL + "?help&__func=SetIntField&field=%s"
+    SetBoolFieldURL   = BaseURL + "?help&__func=SetBoolField&field=%s"
+    GetStringFieldURL = BaseURL + "?help&__func=GetStringField&addr=%s"
+    GetIntFieldURL    = BaseURL + "?help&__func=GetIntField&addr=%s"
+    GetBoolFieldURL   = BaseURL + "?help&__func=GetBoolField&addr=%s"
+    ViewAllFieldsURL  = BaseURL + ":u/%s"
+    ViewFieldURL      = BaseURL + ":f/%s/%s"
+)
+
+func homeHandler(res *mux.ResponseWriter, req *mux.Request) {
+    var b bytes.Buffer
+    addresses := make(map[string]struct{})
+
+    b.WriteString("<html><body>")
+    b.WriteString("<h1>Profiles</h1>")
+    b.WriteString("<h2>Setters</h2>")
```

Comment:
return markdown instead of HTML.

Reply:
https://github.com/gnolang/gno/pull/1983/commits/6755ac92c64d1ba544ba57fc71bd4c58a81799bd